blog/2020-11-05-HowMuchFasterIsJava15.adoc

```diff
@@ -0,0 +1,105 @@
+= How much faster is Java 15?
+:page-interpolate: true
+:awestruct-author: mtomco
+:awestruct-layout: blogPostBase
+:awestruct-tags: [production, benchmark]
+
+Java 15 was released on the 15th of September 2020 and has promised to bring with it a few performance tweaks in its G1GC
+and ParallelGC garbage collectors. What does it mean for
+https://www.optaplanner.org/[OptaPlanner]? Have the improvements made in the G1GC outperformed the reigning champion since Java 8 or does the ParallelGC still
```

Comment:
```suggestion
https://www.optaplanner.org/[OptaPlanner]? Have the improvements made in the G1GC outperform the reigning champion since Java 8 or does the ParallelGC still
```